File: src/point1.php
<?php

class CodeTimer
{
    private float $timeStart;
    private float $timeStop;

    public function __construct(private string $id)
    {
    }

    public function start() : void
    {
        $this->timeStart = microtime(true);
    }

    public function stop() : void
    {
        $this->timeStop = microtime(true);
    }

    public function result() : string
    {
        $time = $this->timeStop - $this->timeStart;
        return "{$this->id}: {$time} sek";
    }
}

$codeTimer = new CodeTimer('Point 1');

$codeTimer->start();

$data = file_get_contents('data/data.json');
$data = json_decode($data);

$codeTimer->stop();

echo $codeTimer->result() . "\n";

File: src/point1.py
import time
import pandas as ps


class CodeTimer:
    _timeStart = 0.0
    _timeStop = 0.0
    _id = ""

    def __init__(self, id):
        self.id = id

    def start(self):
        self._timeStart = time.time()
    
    def stop(self):
        self._timeStop = time.time()

    def result(self):
        timeDiff = self._timeStop - self._timeStart
        return f'{self.id}: {timeDiff} sek'

codeTimer = CodeTimer("Point 1")
codeTimer.start()

data = ps.read_json("data/data.json")

codeTimer.stop()
print(f'{codeTimer.result()}')
